components/ui/AppIntroductoryContent.vue

```diff
@@ -5,9 +5,9 @@
       <p class="app-introductory-content__description">
         {{ description }}
       </p>
-      <AppCta v-if="link" v-bind="link" kind="ghost" />
+      <UiAppCta v-if="link" v-bind="link" kind="ghost" />
```

Comment:
`v-bind` here

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/2980/commits/eef6c40a534c38b7a1e73640346d6f0071d25be8.